Formula/lua@5.1.rb

```diff
@@ -160,7 +160,7 @@ def caveats
 -LUA_A=	liblua.a
 +LUA_A=	liblua.5.1.5.dylib
  CORE_O=	lapi.o lcode.o ldebug.o ldo.o ldump.o lfunc.o lgc.o llex.o lmem.o \
-	lobject.o lopcodes.o lparser.o lstate.o lstring.o ltable.o ltm.o  \
+	lobject.o lopcodes.o lparser.o lstate.o lstring.o ltable.o ltm.o \
```

Comment:
This is a false positive and will break this patch.

Reply:
@bo98 Removed this commit in latest push